### ISSUE TITLE
Problem with generator

### ISSUE THREAD

**Issue by Genkilabs** [stve/bower#8]
Using Ruby 2.2.2 and Rails 4.0.13 I receive an error running the generator after bundling the gem.

```
$ rails generate bower:install
      create  bower_components
Could not find "bower.json" in any of your source paths.
```
I just copied the two files from ```./lib/generators/templates``` into my rails project's root path and its fine. Maybe the generator needs some more explicit pathing?

**Comment by stve** [comment]
Thanks for reporting, I'll have to take a look into it to see what's happening.

**Comment by f3r** [comment]
+1 Thanks!

I also copy the same files `bower.json` and `bowerrc` to `~/.rvm/gems/ruby-2.2.1@wdi/gems/bower-0.0.4/lib/generators/templates` and that did the trick for me

**Comment by stve** [comment]
Please try using version 0.0.5, I changed the glob in the gemspec and it resulted in these files being excluded from the packaged gem. Sorry for the delay in fixing this.

Fixed with e3088727bfe1a1ea5221fb4398ad1f2c7d1135fe
